### DISCUSSION TITLE
WiX v3.x vs v4 & v5 | .wixproj vs .csproj

### DISCUSSION THREAD

**Discussion by BPJDeLozier** [wixtoolset/issues#8806] [Questions] [answered]
We use WiX v3.x to generate a Custom Action dll which is incorporated into an InstallShield project.
We are thinking about updating WiX.

My questions:
WiX can generate a full install package thus not having to use InstallShield?

Is the Visual Studio .wixproj the full install package ... vs what we have right now which is a standard .csproj to produce the Custom Action dll?

**Comment by robmen** [comment, marked as answer]
> WiX can generate a full install package thus not having to use InstallShield?

Yep.

> Is the Visual Studio .wixproj the full install package

Yep, that's the code to create your installation package.

> standard .csproj to produce the Custom Action DLL

You'll still have that for the CustomAction DLL, that gets referenced (and included) by the .wixproj.

**Reply by BPJDeLozier** [reply]
Thank you robmen

**Comment by robmen** [comment]
We leave discussions open since search doesn't seem to find closed ones. Thanks.
